### PR TITLE
Dp link

### PR DESCRIPTION
**Pull request by aditudorache**

### CODE REVIEW

File: src/shared/components/route-link/RouteLinkWrapper.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render } from 'react-dom';
+import { Provider } from 'react-redux';
+
+import RouteLink from './RouteLink';
+
+const RouteLinkWrapper = (props) => (
```
Comment (boris-arkenaar) [inline src/shared/components/route-link/RouteLinkWrapper.jsx]: `src/shared/wrappers/route-link/RouteLinkWrapper.jsx`

File: modules/page/components/homepage/homepage.html [hunk 2]
```diff
@@ -110,6 +110,9 @@ <h1 class="u-padding__left--3 c-homepage__news-header">Nieuws</h1>
                             <dp-link type="SHOW_PAGE"
                                      payload="{name: 'content-overzicht', type: 'news'}"
                                      >Nieuwsoverzicht</dp-link>
+                            <dp-link-react type="SHOW_PAGE"
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply modules/page/components/homepage/homepage.html]: klopt

File: modules/shared/services/redux/application-state.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import * as Redux from 'redux';
+import stateUrlConverter from '../../../../src/shared/services/routing/state-url-converter';
+
+const applicationState = () => {
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply modules/shared/services/redux/application-state.js]: will be done next

File: modules/shared/components/link-react/link-react.component.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                if (routeLinkContainer) {
+                    const props = {
+                        className: vm.className,
+                        inline: vm.inline,
+                        hoverText: vm.hoverText,
+                        type: vm.type,
+                        payload: vm.payload
+                    };
+
+                    const innerText = $element.find('ng-transclude').find('span')[0] &&
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply modules/shared/components/link-react/link-react.component.js]: done

File: modules/atlas/components/dashboard/dashboard-columns.factory.js [hunk 5]
```diff
@@ -118,7 +118,7 @@
                 // Only print the map when detail is NOT fullscreen and has geometry
                 return !state.detail.isFullscreen && angular.isObject(state.detail.geometry);
             } else {
-                return !state.page.name && !state.search;
+                return state.page && !state.page.name && !state.search;
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply modules/atlas/components/dashboard/dashboard-columns.factory.js]: sure! thanks!

File: src/shared/services/util/util.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+export const isFunction = (func) => typeof func === 'function';
+
+export const isDefined = (value) => typeof value !== 'undefined';
+
+export const isString = (value) => typeof value === 'string';
+
+export const cloneObject = (obj) => {
```
Comment (boris-arkenaar) [inline src/shared/services/util/util.js]: Why a clone function? And it seems it is not being used anywhere?

File: src/shared/services/redux/application-state.test.js [hunk 7]
```diff
@@ -3,7 +3,7 @@ import applicationState from './application-state';
 
 jest.mock('redux');
 
-describe('The applicationState', () => {
+fdescribe('The applicationState', () => {
```
Comment (boris-arkenaar) [inline src/shared/services/redux/application-state.test.js]: haal je die `f` wel nog weg?

**Pull request merged**